configure.in

```diff
@@ -50,13 +50,13 @@ fi
 # check we're building against a supported version of PostgreSQL
 citusac_pg_config_version=$($PG_CONFIG --version 2>/dev/null)
 version_num=$(echo "$citusac_pg_config_version"|
-              $SED -e 's/^PostgreSQL \([[0-9]]*\)\.\([[0-9]]*\)\([[a-zA-Z0-9.]]*\)$/\1.\2/')
+              $SED -e 's/^PostgreSQL \([[0-9]]*\)\(\.[[0-9]]*\)\{0,1\}\(.*\)$/\1\2/')
```

Comment:
I think this comment should probably be on the actual test a few lines down, since this regex should match `10.0` just fine. I'll add a `10.0` case to the test a few lines down to permit `10.0` once it's released (is that sufficient?).